### PR TITLE
Extended keyboard actuator

### PR DESCRIPTION
**Pull request by cehberlin**

### CODE REVIEW

**Comment by severin-lemaignan** [comment]
Thanks for your contribution. Before we review it, could you please rebase your pull request on the HEAD of master?

**Comment by cehberlin** [comment]
Hi, I tried to rebase it, but somehow messed it up. Can you provide me help on how to do this? Or maybe just merge the one file I changed, should not be difficult, since the file has not changed for long time

**Comment by severin-lemaignan** [comment]
``git rebase master`` in your ``extended_keyboard_actuator`` branch should do the job...

File: src/morse/actuators/keyboard.py [hunk 1]
```diff
@@ -14,6 +14,10 @@ class Keyboard(Actuator):
 
     :kbd:`Up` forward
     :kbd:`Down` backwards
+    :kbd:`J` left
+    :kbd:`L` left
```
Comment (pierriko) [inline src/morse/actuators/keyboard.py]: should be "right"

**Comment by cehberlin** [comment]
I have created a new clean request and extended #657. Sorry for the inconvenience caused.

**Comment by severin-lemaignan** [comment]
Superseed by #657. Closing.

**Pull request closed without merging**